commons-numbers-complex-arrays/src/main/java/org/apache/commons/numbers/complex/arrays/ComplexList.java

```diff
@@ -348,4 +433,25 @@ public void replaceAll(ComplexUnaryOperator<Void> operator) {
         }
         modCount++;
     }
+
+    /**
+     * Replaces each element of the list with the result of applying the action to that element.
+     *
+     * @param action The action to apply to each element.
+     */
+    public void forEach(ComplexConsumer action) {
+        Objects.requireNonNull(action);
+        final double[] parts = this.realAndImagParts;
+        final int m = size;
+        final int expectedModCount = modCount;
```

Comment:
The mod count is increased when the list is structurally modified. Here no modifications are occurring so this can be removed.